### ISSUE TITLE
Relative module URLs starting without "./"

### ISSUE THREAD

**Issue by bangseongbeom** [whatwg/loader#131]
I agree that the name resolution policy may not follow the URL syntax fully, but it may confuse the users if it's invalid to start without "./" because `<link>` in HTML and `@import` in CSS permit both with and without.

**Comment by domenic** [comment]
> and it is specified here: https://html.spec.whatwg.org/#resolve-a-module-specifier

Not quite, that specifies how `import` is processed. I believe you're looking for https://html.spec.whatwg.org/#prepare-a-script step 16.4.

**Comment by caridy** [comment]
:+1:
